apps/base-docs/docs/tools/onboarding.md

```diff
@@ -54,6 +55,12 @@ You can [get started with Privy here](https://docs.privy.io/guide/quickstart), a
 
 ---
 
+## Sequence
+
+[Sequence](https://sequence.xyz/) is the leading all-in-one development platform for integrating web3 into games. Onboard, monetize, grow, and retain players with Sequence’s award-winning technology including: non-custodial Embedded Wallets, white labeled marketplaces and marketplace API, indexer, relayer, node gateway, Unity/Unreal/React Native/Mobile SDKs, transaction API, contract deployment, analytics, and more. [Learn more here](https://sequence.xyz/) and start creating on [Sequence Builder](https://sequence.build/) now.
```

Comment:
Could you please drop the superlative (s/a leading/the leading)?

Reply:
@wbnns Thank you for the quick review! Both changes are added now.